### ISSUE TITLE
why does it get flagged as ransomware?

### ISSUE THREAD

**Issue by aaaaaaaaaaaaaaaaaeeeeee** [Endermanch/XPKeygen#36]
i know false positives exist, but why ransomware? did all the antivirus companies came together and look at this program and say "hmm a windows xp key generator, that can't be good, hmm well we need to flag it as something malicious, ehh we'll just flag it as ransomware for no reason" LIKE WTF i ran it in a sandbox and it gave a threat score of 1/10. 

so either microsoft is bribing these antivirus companies to block this app bc they don't like it, or are they actually dumb.

![Capture](https://github.com/user-attachments/assets/8fe9f1f6-47cb-4f42-85be-7663e516afa1)

i just don't understand, the program in no way behaves like ransomware, also why tf is the community vote so low??

**Comment by aaaaaaaaaaaaaaaaaeeeeee** [comment]
maybe the community score is so low bc the keys don't work and people are mad?

**Comment by thepwrtank18** [comment]
Can you send the link to the VirusTotal result?

**Comment by aaaaaaaaaaaaaaaaaeeeeee** [comment]
> Can you send the link to the VirusTotal result?

yeah here:https://www.virustotal.com/gui/file/875bcac02aaad8547c48a2d38436ed62607a317c7a9407384550fe0606268ad4

**Comment by thepwrtank18** [comment]
![image](https://github.com/user-attachments/assets/253360f4-f1ec-40a2-859e-00f32e64c078)
It's literally 1 person that put their entire body weight on the scale.

Some people are more equal than others, i guess

**Comment by aaaaaaaaaaaaaaaaaeeeeee** [comment]
> ###
> **[thepwrtank18](/thepwrtank18) ** commented [Aug 16, 2024](#issuecomment-2294145545) •

that's a hater right there lol, but it's still intresting why more then half of antiviruses flag it as "ransomware", ik it would get flagged for something but i wasn't expecting ransomware 💀 , maybe bc of the name (key gen in it's name) but surely the name can't be enough.

**Comment by DEMnetwork** [comment]
I think the security vendors think that every key/license key generator
wants to steal your OS.

**Comment by DayreaverDoesThings** [comment]
> ![image](https://private-user-images.githubusercontent.com/57580668/358701237-253360f4-f1ec-40a2-859e-00f32e64c078.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNzk3ODgsIm5iZiI6MTcyNTM3OTQ4OCwicGF0aCI6Ii81NzU4MDY2OC8zNTg3MDEyMzctMjUzMzYwZjQtZjFlYy00MGEyLTg1OWUtMDBmMzJlNjRjMDc4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDE2MDQ0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2OTgxZmU5M2EzZWVkNjllNDM1YTkwMWVjY2VjYmYxZDlkNzRiYTdhZGE5NWU2NzRmNmMwODE4YjI5NWJhMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.z4Gl9zZI32YjHy7SqTsGfU3Tl9_VxLghbteeqlg6zW8)

lmao yeah
it seems trolls have litterally no chill

**Comment by Bang1338** [comment]
> but it's still intresting why more then half of antiviruses flag it as "ransomware"

because most of ransomware using RSA algorithm to... ransom user, of course.
